src/app/shared/config.ts

```diff
@@ -169,12 +169,12 @@ function getBaseUrl() {
       : 'https://api.ctemplar.com/';
   }
   if (AppConfig.local) {
-    return 'http://localhost:8000/';
+    return 'http://10.70.3.105:8000/';
```

Comment:
@superdev0714 
I think this change wasn't intended. 

If this way is giving headache, I suggest setting API URL based on the environment file, so every developer can override their local one. It is the first point in #932, FYI if you want to take a look.

Reply:
I don't need it right now, I usually develop on a machine, but sometimes in special cases.
I will revert this part.